CHANGELOG.md

```diff
@@ -6,6 +6,53 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [30.0.0]
+### Added
+- **BREAKING:** Introduce `getNetworkState`, `getPreferencesState`, and `onTokenListStateChange` to the TokenDetectionController constructor options object ([#808](https://github.com/MetaMask/controllers/pull/808))
+  - getPreferencesState provides the default value for `useTokenDetection` from the PreferencesController state.
+  - getNetworkState provides the default value for `chainId` from the NetworkController state.
+  - onTokenListStateChange listener triggers a detect action whenever the TokenListController finishes fetching a token list.
+- **BREAKING:** Expose `chainId` to AssetsContractController and add check whether current network supports token detection ([#809](https://github.com/MetaMask/controllers/pull/809))
+  - Consumers will need to pass listener method `onNetworkStateChange` in `AssetsContractController` constructor options object. This method should be called with network/provider details when network changes occur.
+- Add `onCollectibleAdded` event handler to the CollectiblesController constructor ([#814](https://github.com/MetaMask/controllers/pull/814))
+  - This event handler was added to allow us to capture metrics.
+
+### Changed
+- **BREAKING:** Change `addTokens` to `addDetectedTokens` on the TokenDetectionController constructor ([#808](https://github.com/MetaMask/controllers/pull/808))
```

Comment:
Wording: "Rename `addTokens` on the TokenDetectionController constructor options object to `addDetectedTokens`"?

Reply:
updated here: https://github.com/MetaMask/controllers/pull/848/commits/8f0b66e74d974e5e1da4d250f99b289e2eff70c3